bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -1,11 +1,14 @@
 import Foundation
 
 private func dummy() {
-    _ = NSLocalizedString("'%s' isn't yet supported on WordPress for Android", comment: "translators: %s: Name of the block")
-    _ = NSLocalizedString("'%s' isn't yet supported on WordPress for iOS", comment: "")
+    _ = NSLocalizedString("%1$s. Current value is %2$s", comment: "translators: accessibility text. Inform about current value. %1$s: Control label %2$s: Current value.")
```

Comment:
@koke do we still need to make additions to this file?

Reply:
I think that's still the case, @Tug can you confirm?

Reply:
Yes, but those should only be updated when we make a release though. Not sure why we're updating `bundle/` in this PR?

Reply:
I was publishing `bundle` to create a testing build in WordPress-iOS for Thomas. Will revert these changes.